components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -116,7 +118,41 @@ public void addOAuthApplication(OAuthAppDO consumerAppDO) throws IdentityOAuthAd
             prepStmt.setString(3, username);
             prepStmt.setInt(4, tenantId);
             // it is assumed that the OAuth version is 1.0a because this is required with OAuth 1.0a
-            prepStmt.setString(5, OAuthConstants.OAuthVersions.VERSION_1A);
```

Comment:
Looks like we are duplicating this method for API backward compatibility. Since this is a new carbon-identity version (at least no products have gone yet) can we change existing method ?